src/zope/component/tests/test__declaration.py

```diff
@@ -80,22 +80,11 @@ def _run_generated_code(self, code, globs, locs,
                             fails_under_py3k=True,
                            ):
         import warnings
-        #from zope.component._compat import PYTHON3
-        PYTHON3 = False
```

Comment:
That's true for the `zope.interface` versions---they specifically blacklist against that. But `zope.component.adapts` can be used in Python 3:

```pycon
Python 3.5.3 (default, Apr 23 2017, 18:09:27)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from zope.component._declaration import adapts
>>> from zope.interface import Interface
>>> class Foo(object):
...   adapts(Interface)
...
>>> from zope.component._declaration import adaptedBy
>>> adaptedBy(Foo)
(<InterfaceClass zope.interface.Interface>,)
```

Which is the same as under Python 2:
```pycon
Python 2.7.13 (default, May  5 2017, 09:16:47)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from zope.component._declaration import adapts
>>> from zope.interface import Interface
>>> class Foo(object):
...   adapts(Interface)
...
>>> from zope.component._declaration import adaptedBy
>>> adaptedBy(Foo)
(<InterfaceClass zope.interface.Interface>,)
```

So these tests all pass both places.